css/style.css

```diff
@@ -1,19 +1,280 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  width: 100%;
 }
 
 /**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+    * Add your custom styles below
+    * 
+    * Remember: 
+    * - Be organised, use comments and separate your styles into meaningful chunks
+    *    for example: General styles, Navigation styles, Hero styles, Footer etc.
+    * 
+    * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
+    */
+
+/*--HEADER--*/
+
+.header-container {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  height: 30px;
+  width: 30px;
+  padding: 10px 0 15px 10px;
+}
+
+.hamburger {
+  height: 20px;
```

Comment:
it is better practice to use relative units rather than pixels for sizing things like this - e.g. `2rem`. This keeps them relatively sized if the user were to change their font size in the browser for example